tests/foreman/api/test_hostgroup.py

```diff
@@ -198,6 +198,24 @@ def test_positive_create_with_name(self):
                 ).create()
                 self.assertEqual(name, hostgroup.name)
 
+    @tier1
+    def test_positive_clone(self):
+        """Create a hostgroup by cloning an existing one
+
+        @id: 44ac8b3b-9cb0-4a9e-ad9b-2c67b2411922
+
+        @assert: A hostgroup is cloned with new name
+        """
+        hostgroup = entities.HostGroup(
+            location=[self.loc],
+            organization=[self.org],
+        ).create()
+        hostgroup_cloned_name = gen_string('alpha')
+        hostgroup_cloned = entities.HostGroup(
+            id=hostgroup.id
+        ).clone(data={'name': hostgroup_cloned_name})
+        self.assertEqual(hostgroup_cloned['name'], hostgroup_cloned_name)
```

Comment:
This has nothing to do with this PR but... can we file an issue to improve this test by asserting that the entire object is equal to the cloned one? Checking that only the name matches assumes that everything else also matches and I don't think we should make assumptions.